RadarSDK/RadarContext+Internal.h

```diff
@@ -0,0 +1,26 @@
+//
+//  RadarContext+Internal.h
+//  RadarSDK
+//
+//  Created by Cory Pisano on 1/30/20.
+//  Copyright © 2020 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RadarContext.h"
+#import "RadarGeofence.h"
+#import "RadarUserInsights.h"
+
+@interface RadarContext ()
+
+- (instancetype _Nullable)initWithLive:(BOOL)live
+                             geofences:(NSArray * _Nullable)geofences
```

Comment:
I don't think we need `_Nullable` for geofences - we're returning an empty array, even if no geofences are found